storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw_test.go

```diff
@@ -308,81 +301,79 @@ func TestFromMetrics(t *testing.T) {
 				generateAttributes(point.Attributes(), "series", 2)
 				curTs = curTs.Add(defaultLookbackDelta / 4)
 			}
+		}
 
-			namer := otlptranslator.MetricNamer{}
-			name, err := namer.Build(TranslatorMetricFromOtelMetric(m))
-			require.NoError(t, err)
-			expMetadata = append(expMetadata, prompb.MetricMetadata{
-				Type:             otelMetricTypeToPromMetricType(m),
-				MetricFamilyName: name,
-				Help:             m.Description(),
-				Unit:             m.Unit(),
-			})
+		mockAppender := &mockCombinedAppender{}
+		converter := NewPrometheusConverter(mockAppender)
+		settings := Settings{
+			LookbackDelta: defaultLookbackDelta,
 		}
 
-		converter := NewPrometheusConverter()
-		annots, err := converter.FromMetrics(
-			context.Background(),
-			request.Metrics(),
-			Settings{
-				LookbackDelta: defaultLookbackDelta,
-			},
-		)
+		annots, err := converter.FromMetrics(context.Background(), request.Metrics(), settings)
```

Comment:
[Nit] An unnecessary stylistic change, that makes reviewing more difficult:
```suggestion
		annots, err := converter.FromMetrics(
			context.Background(),
			request.Metrics(),
			Settings{
				LookbackDelta: defaultLookbackDelta,
			},
		)
```